Regression in 0.10.3: SyntaxError during

Upgrading responses from 0.10.2 to 0.10.3 causes pytest's test discovery to fail with the following error:

```
==================================== ERRORS ====================================
_____________________ ERROR collecting tests/test_foo.py ______________________
…/python3.6/site-packages/_pytest/python.py:450: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
…/python3.6/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
<frozen importlib._bootstrap>:971: in _find_and_load
    ???
<frozen importlib._bootstrap>:955: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:656: in _load_unlocked
    ???
<frozen importlib._bootstrap>:626: in _load_backward_compatible
    ???
…/python3.6/site-packages/_pytest/assertion/rewrite.py:294: in load_module
    six.exec_(co, mod.__dict__)
tests/test_foo.py:1234: in <module>
    class TestFoo(TestCase):
tests/test_foo.py:1345: in TestFoo
    def test_foo(self) -> None:
…/python3.6/site-packages/responses.py:553: in activate
    return get_wrapped(func, self)
…/python3.6/site-packages/responses.py:158: in get_wrapped
    evaldict,
E     File "<string>", line 3
E       return func(self) -> None
E                          ^
E   SyntaxError: invalid syntax
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 1 error in 6.21 seconds ============================
```

Reverting back to 0.10.2 avoids the error.

--- a/responses.py
+++ b/responses.py
@@ -149,7 +149,12 @@ def get_wrapped(func, responses):
                 inspect.Parameter("args", inspect.Parameter.VAR_POSITIONAL),
                 inspect.Parameter("kwargs", inspect.Parameter.VAR_KEYWORD),
             ]
-            signature = signature.replace(parameters=wrapper_params)
+        else:
+            wrapper_params = [
+                param.replace(annotation=inspect.Parameter.empty)
+                for param in signature.parameters.values()
+            ]
+        signature = signature.replace(parameters=wrapper_params)
 
         wrapper_args = str(signature)
         params_without_defaults = [

--- a/test_responses.py
+++ b/test_responses.py
@@ -560,7 +560,7 @@ def test_activate_doesnt_change_signature_with_return_type():
     # Add type annotations as they are syntax errors in py2.
     # Use a class to test for import errors in evaled code.
     test_function.__annotations__["return"] = Mock
-    test_function.__annotations__["a"] = str
+    test_function.__annotations__["a"] = Mock
 
     decorated_test_function = responses.activate(test_function)
     if hasattr(inspect, "signature"):

